chess-AI/train.py

```diff
@@ -155,22 +158,46 @@ def training_episode(self, nnet, games, epochs, batch_size):
 
                         # Reset gradients
                         opt.zero_grad()
-
                 print(losses)
-
+            # Saves model to specified file, or a new file if not specified.
+            if self.save_path != None:
```

Comment:
`if self.save_path is not None`, see [style guide](https://google.github.io/styleguide/pyguide.html#2144-decision)